{"https://finance.yahoo.com/quote/": {"count": 55, "('Connection aborted.', RemoteDisconnected('Remote end closed connection without response'))": 1, "HTTPSConnectionPool(host='finance.yahoo.com', port=443): Max retries exceeded with url: /quote/CB/history (Caused by NewConnectionError('<urllib3.connection.HTTPSConnection object at 0x00000279D26EF490>: Failed to establish a new connection: [Errno 11001] getaddrinfo failed'))": 1, "HTTPSConnectionPool(host='finance.yahoo.com', port=443): Max retries exceeded with url: /quote/PLD/history (Caused by NewConnectionError('<urllib3.connection.HTTPSConnection object at 0x00000279D26EF490>: Failed to establish a new connection: [Errno 11001] getaddrinfo failed'))": 1, "HTTPSConnectionPool(host='finance.yahoo.com', port=443): Max retries exceeded with url: /quote/ADP/history (Caused by NewConnectionError('<urllib3.connection.HTTPSConnection object at 0x00000279D25AB700>: Failed to establish a new connection: [Errno 11001] getaddrinfo failed'))": 1, "('Connection aborted.', ConnectionResetError(10054, 'An existing connection was forcibly closed by the remote host', None, 10054, None))": 5, "list index out of range": 14, "('Connection aborted.', TimeoutError(10060, 'A connection attempt failed because the connected party did not properly respond after a period of time, or established connection failed because connected host has failed to respond', None, 10060, None))": 2, "HTTPSConnectionPool(host='finance.yahoo.com', port=443): Max retries exceeded with url: /quote/P911.DE/history (Caused by NewConnectionError('<urllib3.connection.HTTPSConnection object at 0x00000279D3098580>: Failed to establish a new connection: [WinError 10060] A connection attempt failed because the connected party did not properly respond after a period of time, or established connection failed because connected host has failed to respond'))": 1, "HTTPSConnectionPool(host='finance.yahoo.com', port=443): Max retries exceeded with url: /quote/EQNR/history (Caused by NewConnectionError('<urllib3.connection.HTTPSConnection object at 0x00000279D6690970>: Failed to establish a new connection: [Errno 11001] getaddrinfo failed'))": 1, "HTTPSConnectionPool(host='finance.yahoo.com', port=443): Max retries exceeded with url: /quote/ZTS/history (Caused by NewConnectionError('<urllib3.connection.HTTPSConnection object at 0x00000279D1CCC5E0>: Failed to establish a new connection: [Errno 11001] getaddrinfo failed'))": 1, "HTTPSConnectionPool(host='finance.yahoo.com', port=443): Max retries exceeded with url: /quote/EQIX/history (Caused by NewConnectionError('<urllib3.connection.HTTPSConnection object at 0x00000279D37EF310>: Failed to establish a new connection: [Errno 11001] getaddrinfo failed'))": 1, "HTTPSConnectionPool(host='finance.yahoo.com', port=443): Max retries exceeded with url: /quote/CP/history (Caused by NewConnectionError('<urllib3.connection.HTTPSConnection object at 0x00000279D37EF850>: Failed to establish a new connection: [Errno 11001] getaddrinfo failed'))": 1, "HTTPSConnectionPool(host='finance.yahoo.com', port=443): Max retries exceeded with url: /quote/8031.T/history (Caused by NewConnectionError('<urllib3.connection.HTTPSConnection object at 0x00000279D50A7520>: Failed to establish a new connection: [Errno 11001] getaddrinfo failed'))": 1, "HTTPSConnectionPool(host='finance.yahoo.com', port=443): Max retries exceeded with url: /quote/WMMVF/history (Caused by NewConnectionError('<urllib3.connection.HTTPSConnection object at 0x00000279D5940D90>: Failed to establish a new connection: [Errno 11001] getaddrinfo failed'))": 1, "Can't fetch current ip, even after retrying... Check your internet connection.": 1, "HTTPSConnectionPool(host='finance.yahoo.com', port=443): Max retries exceeded with url: /quote/LRCX/history (Caused by NewConnectionError('<urllib3.connection.HTTPSConnection object at 0x00000279D5D68640>: Failed to establish a new connection: [Errno 11001] getaddrinfo failed'))": 1, "HTTPSConnectionPool(host='finance.yahoo.com', port=443): Max retries exceeded with url: /quote/601328.SS/history (Caused by NewConnectionError('<urllib3.connection.HTTPSConnection object at 0x00000279CFF790D0>: Failed to establish a new connection: [Errno 11001] getaddrinfo failed'))": 1, "HTTPSConnectionPool(host='finance.yahoo.com', port=443): Max retries exceeded with url: /quote/APO/history (Caused by NewConnectionError('<urllib3.connection.HTTPSConnection object at 0x00000279CFF79A90>: Failed to establish a new connection: [Errno 11001] getaddrinfo failed'))": 1, "HTTPSConnectionPool(host='finance.yahoo.com', port=443): Max retries exceeded with url: /quote/MNST/history (Caused by NewConnectionError('<urllib3.connection.HTTPSConnection object at 0x00000279CF916820>: Failed to establish a new connection: [Errno 11001] getaddrinfo failed'))": 1, "(\"Connection broken: ConnectionAbortedError(10053, 'An established connection was aborted by the software in your host machine', None, 10053, None)\", ConnectionAbortedError(10053, 'An established connection was aborted by the software in your host machine', None, 10053, None))": 2, "[Errno 13] Permission denied: 'C:\\\\Users\\\\Burt\\\\Documents\\\\Git\\\\podcast\\\\stocks_history.xlsx'": 1, "HTTPSConnectionPool(host='finance.yahoo.com', port=443): Max retries exceeded with url: /quote/SU/history (Caused by NewConnectionError('<urllib3.connection.HTTPSConnection object at 0x0000025483FA8610>: Failed to establish a new connection: [Errno 11001] getaddrinfo failed'))": 1, "HTTPSConnectionPool(host='finance.yahoo.com', port=443): Max retries exceeded with url: /quote/MCHP/history (Caused by NewConnectionError('<urllib3.connection.HTTPSConnection object at 0x0000025483FA8520>: Failed to establish a new connection: [Errno 11001] getaddrinfo failed'))": 1, "HTTPSConnectionPool(host='finance.yahoo.com', port=443): Max retries exceeded with url: /quote/4519.T/history (Caused by NewConnectionError('<urllib3.connection.HTTPSConnection object at 0x0000025483FA8400>: Failed to establish a new connection: [Errno 11001] getaddrinfo failed'))": 1, "HTTPSConnectionPool(host='finance.yahoo.com', port=443): Max retries exceeded with url: /quote/PLTR/history (Caused by NewConnectionError('<urllib3.connection.HTTPSConnection object at 0x0000025480F426A0>: Failed to establish a new connection: [Errno 11001] getaddrinfo failed'))": 1, "HTTPSConnectionPool(host='finance.yahoo.com', port=443): Max retries exceeded with url: /quote/CM/history (Caused by NewConnectionError('<urllib3.connection.HTTPSConnection object at 0x00000254FD68D910>: Failed to establish a new connection: [WinError 10060] A connection attempt failed because the connected party did not properly respond after a period of time, or established connection failed because connected host has failed to respond'))": 1, "HTTPSConnectionPool(host='finance.yahoo.com', port=443): Max retries exceeded with url: /quote/RR.L/history (Caused by NewConnectionError('<urllib3.connection.HTTPSConnection object at 0x00000254861F99D0>: Failed to establish a new connection: [Errno 11001] getaddrinfo failed'))": 1, "HTTPSConnectionPool(host='finance.yahoo.com', port=443): Max retries exceeded with url: /quote/PAYX/history (Caused by NewConnectionError('<urllib3.connection.HTTPSConnection object at 0x00000254861F9A00>: Failed to establish a new connection: [Errno 11001] getaddrinfo failed'))": 1, "HTTPSConnectionPool(host='finance.yahoo.com', port=443): Max retries exceeded with url: /quote/CS.PA/history (Caused by NewConnectionError('<urllib3.connection.HTTPSConnection object at 0x000001D5F2A7A160>: Failed to establish a new connection: [Errno 11001] getaddrinfo failed'))": 1, "HTTPSConnectionPool(host='finance.yahoo.com', port=443): Max retries exceeded with url: /quote/SBER.ME/history (Caused by NewConnectionError('<urllib3.connection.HTTPSConnection object at 0x000001D5F13E14C0>: Failed to establish a new connection: [Errno 11001] getaddrinfo failed'))": 1, "HTTPSConnectionPool(host='finance.yahoo.com', port=443): Max retries exceeded with url: /quote/BCS/history (Caused by NewConnectionError('<urllib3.connection.HTTPSConnection object at 0x000001CCA96C9910>: Failed to establish a new connection: [WinError 10060] A connection attempt failed because the connected party did not properly respond after a period of time, or established connection failed because connected host has failed to respond'))": 1, "HTTPSConnectionPool(host='finance.yahoo.com', port=443): Max retries exceeded with url: /quote/600309.SS/history (Caused by NewConnectionError('<urllib3.connection.HTTPSConnection object at 0x000001CCAB18D3A0>: Failed to establish a new connection: [Errno 11001] getaddrinfo failed'))": 1, "HTTPSConnectionPool(host='finance.yahoo.com', port=443): Max retries exceeded with url: /quote/6857.T/history (Caused by NewConnectionError('<urllib3.connection.HTTPSConnection object at 0x0000019B0665C670>: Failed to establish a new connection: [WinError 10060] A connection attempt failed because the connected party did not properly respond after a period of time, or established connection failed because connected host has failed to respond'))": 1, "HTTPSConnectionPool(host='finance.yahoo.com', port=443): Max retries exceeded with url: /quote/NET/history (Caused by NewConnectionError('<urllib3.connection.HTTPSConnection object at 0x0000019B06D717F0>: Failed to establish a new connection: [Errno 11001] getaddrinfo failed'))": 1, "HTTPSConnectionPool(host='finance.yahoo.com', port=443): Max retries exceeded with url: /quote/EVO.ST/history (Caused by NewConnectionError('<urllib3.connection.HTTPSConnection object at 0x0000019B065DA7F0>: Failed to establish a new connection: [WinError 10060] A connection attempt failed because the connected party did not properly respond after a period of time, or established connection failed because connected host has failed to respond'))": 1, "HTTPSConnectionPool(host='finance.yahoo.com', port=443): Max retries exceeded with url: /quote/PKN.WA/history (Caused by NewConnectionError('<urllib3.connection.HTTPSConnection object at 0x0000019B0A776C70>: Failed to establish a new connection: [WinError 10060] A connection attempt failed because the connected party did not properly respond after a period of time, or established connection failed because connected host has failed to respond'))": 1}, "https://www.google.com/finance/quote/": {"count": 73, "HTTPSConnectionPool(host='www.google.com', port=443): Max retries exceeded with url: /finance/quote/2222:TADAWUL (Caused by NewConnectionError('<urllib3.connection.HTTPSConnection object at 0x000001C478E003D0>: Failed to establish a new connection: [Errno 11001] getaddrinfo failed'))": 1, "No connection adapters were found for 'V:NYSE'": 2, "(\"Connection broken: ConnectionAbortedError(10053, 'An established connection was aborted by the software in your host machine', None, 10053, None)\", ConnectionAbortedError(10053, 'An established connection was aborted by the software in your host machine', None, 10053, None))": 1, "No connection adapters were found for 'T:NYSE'": 1, "HTTPSConnectionPool(host='www.google.com', port=443): Max retries exceeded with url: /finance/quote/LMT:NYSE (Caused by NewConnectionError('<urllib3.connection.HTTPSConnection object at 0x000001ABCC663040>: Failed to establish a new connection: [Errno 11001] getaddrinfo failed'))": 1, "HTTPSConnectionPool(host='www.google.com', port=443): Max retries exceeded with url: /finance/quote/BSX:NYSE (Caused by NewConnectionError('<urllib3.connection.HTTPSConnection object at 0x000001ABCC6BDA30>: Failed to establish a new connection: [WinError 10060] A connection attempt failed because the connected party did not properly respond after a period of time, or established connection failed because connected host has failed to respond'))": 1, "HTTPSConnectionPool(host='www.google.com', port=443): Max retries exceeded with url: /finance/quote/BA:NYSE (Caused by NewConnectionError('<urllib3.connection.HTTPSConnection object at 0x000001ABCC6B7760>: Failed to establish a new connection: [WinError 10060] A connection attempt failed because the connected party did not properly respond after a period of time, or established connection failed because connected host has failed to respond'))": 1, "('Connection aborted.', TimeoutError(10060, 'A connection attempt failed because the connected party did not properly respond after a period of time, or established connection failed because connected host has failed to respond', None, 10060, None))": 3, "HTTPSConnectionPool(host='www.google.com', port=443): Max retries exceeded with url: /finance/quote/PANW:NASDAQ (Caused by SSLError(SSLZeroReturnError(6, 'TLS/SSL connection has been closed (EOF) (_ssl.c:1129)')))": 1, "HTTPSConnectionPool(host='www.google.com', port=443): Max retries exceeded with url: /finance/quote/SPGI:NYSE (Caused by NewConnectionError('<urllib3.connection.HTTPSConnection object at 0x000002495DCF6DF0>: Failed to establish a new connection: [WinError 10060] A connection attempt failed because the connected party did not properly respond after a period of time, or established connection failed because connected host has failed to respond'))": 1, "HTTPSConnectionPool(host='www.google.com', port=443): Max retries exceeded with url: /finance/quote/T:NYSE (Caused by NewConnectionError('<urllib3.connection.HTTPSConnection object at 0x000002495DCEF700>: Failed to establish a new connection: [WinError 10060] A connection attempt failed because the connected party did not properly respond after a period of time, or established connection failed because connected host has failed to respond'))": 1, "HTTPSConnectionPool(host='www.google.com', port=443): Max retries exceeded with url: /finance/quote/3690:KHG (Caused by SSLError(SSLZeroReturnError(6, 'TLS/SSL connection has been closed (EOF) (_ssl.c:1129)')))": 1, "HTTPSConnectionPool(host='www.google.com', port=443): Max retries exceeded with url: /finance/quote/KKR:NYSE (Caused by SSLError(SSLZeroReturnError(6, 'TLS/SSL connection has been closed (EOF) (_ssl.c:1129)')))": 1, "HTTPSConnectionPool(host='www.google.com', port=443): Max retries exceeded with url: /finance/quote/CMG:NYSE (Caused by NewConnectionError('<urllib3.connection.HTTPSConnection object at 0x000002495DE4CBE0>: Failed to establish a new connection: [WinError 10060] A connection attempt failed because the connected party did not properly respond after a period of time, or established connection failed because connected host has failed to respond'))": 1, "Can't fetch current ip, even after retrying... Check your internet connection.": 4, "HTTPSConnectionPool(host='www.google.com', port=443): Max retries exceeded with url: /finance/quote/ATD:TSE (Caused by NewConnectionError('<urllib3.connection.HTTPSConnection object at 0x000002495DA4A4C0>: Failed to establish a new connection: [Errno 11001] getaddrinfo failed'))": 1, "HTTPSConnectionPool(host='www.google.com', port=443): Max retries exceeded with url: /finance/quote/KDP:NASDAQ (Caused by NewConnectionError('<urllib3.connection.HTTPSConnection object at 0x000002495D983AC0>: Failed to establish a new connection: [Errno 11001] getaddrinfo failed'))": 1, "HTTPSConnectionPool(host='www.google.com', port=443): Max retries exceeded with url: /finance/quote/BBRI:IDX (Caused by SSLError(SSLZeroReturnError(6, 'TLS/SSL connection has been closed (EOF) (_ssl.c:1129)')))": 1, "HTTPSConnectionPool(host='www.google.com', port=443): Max retries exceeded with url: /finance/quote/PAYX:NASDAQ (Caused by NewConnectionError('<urllib3.connection.HTTPSConnection object at 0x000002495FCA55B0>: Failed to establish a new connection: [WinError 10060] A connection attempt failed because the connected party did not properly respond after a period of time, or established connection failed because connected host has failed to respond'))": 1, "HTTPSConnectionPool(host='www.google.com', port=443): Max retries exceeded with url: /finance/quote/A:NYSE (Caused by NewConnectionError('<urllib3.connection.HTTPSConnection object at 0x000002495FCA6A00>: Failed to establish a new connection: [Errno 11001] getaddrinfo failed'))": 1, "HTTPSConnectionPool(host='www.google.com', port=443): Max retries exceeded with url: /finance/quote/CCI:NYSE (Caused by NewConnectionError('<urllib3.connection.HTTPSConnection object at 0x000002495FCA5B20>: Failed to establish a new connection: [Errno 11001] getaddrinfo failed'))": 1, "HTTPSConnectionPool(host='www.google.com', port=443): Max retries exceeded with url: /finance/quote/ADS:ETR (Caused by NewConnectionError('<urllib3.connection.HTTPSConnection object at 0x000002495FCA53A0>: Failed to establish a new connection: [Errno 11001] getaddrinfo failed'))": 1, "HTTPSConnectionPool(host='www.google.com', port=443): Max retries exceeded with url: /finance/quote/ALC:NYSE (Caused by NewConnectionError('<urllib3.connection.HTTPSConnection object at 0x000002495E6299A0>: Failed to establish a new connection: [WinError 10060] A connection attempt failed because the connected party did not properly respond after a period of time, or established connection failed because connected host has failed to respond'))": 1, "HTTPSConnectionPool(host='www.google.com', port=443): Max retries exceeded with url: /finance/quote/ARES:NYSE (Caused by NewConnectionError('<urllib3.connection.HTTPSConnection object at 0x000002495E6422B0>: Failed to establish a new connection: [WinError 10060] A connection attempt failed because the connected party did not properly respond after a period of time, or established connection failed because connected host has failed to respond'))": 1, "HTTPSConnectionPool(host='www.google.com', port=443): Max retries exceeded with url: /finance/quote/KHC:NASDAQ (Caused by NewConnectionError('<urllib3.connection.HTTPSConnection object at 0x000002495E62F040>: Failed to establish a new connection: [Errno 11001] getaddrinfo failed'))": 1, "HTTPSConnectionPool(host='www.google.com', port=443): Max retries exceeded with url: /finance/quote/TTD:NASDAQ (Caused by NewConnectionError('<urllib3.connection.HTTPSConnection object at 0x000002495D9C4FD0>: Failed to establish a new connection: [WinError 10060] A connection attempt failed because the connected party did not properly respond after a period of time, or established connection failed because connected host has failed to respond'))": 1, "HTTPSConnectionPool(host='www.google.com', port=443): Max retries exceeded with url: /finance/quote/CHTR:NASDAQ (Caused by NewConnectionError('<urllib3.connection.HTTPSConnection object at 0x000002495E2C3FD0>: Failed to establish a new connection: [WinError 10060] A connection attempt failed because the connected party did not properly respond after a period of time, or established connection failed because connected host has failed to respond'))": 1, "HTTPSConnectionPool(host='www.google.com', port=443): Max retries exceeded with url: /finance/quote/MSCI:NYSE (Caused by NewConnectionError('<urllib3.connection.HTTPSConnection object at 0x000002495DFA7670>: Failed to establish a new connection: [WinError 10060] A connection attempt failed because the connected party did not properly respond after a period of time, or established connection failed because connected host has failed to respond'))": 1, "HTTPSConnectionPool(host='www.google.com', port=443): Max retries exceeded with url: /finance/quote/STM:NYSE (Caused by NewConnectionError('<urllib3.connection.HTTPSConnection object at 0x000002495D9CD850>: Failed to establish a new connection: [WinError 10060] A connection attempt failed because the connected party did not properly respond after a period of time, or established connection failed because connected host has failed to respond'))": 1, "HTTPSConnectionPool(host='www.google.com', port=443): Max retries exceeded with url: /finance/quote/CVE:NYSE (Caused by SSLError(SSLZeroReturnError(6, 'TLS/SSL connection has been closed (EOF) (_ssl.c:1129)')))": 1, "HTTPSConnectionPool(host='www.google.com', port=443): Max retries exceeded with url: /finance/quote/ABEV:NYSE (Caused by SSLError(SSLZeroReturnError(6, 'TLS/SSL connection has been closed (EOF) (_ssl.c:1129)')))": 1, "HTTPSConnectionPool(host='www.google.com', port=443): Max retries exceeded with url: /finance/quote/6503:TYO (Caused by NewConnectionError('<urllib3.connection.HTTPSConnection object at 0x000002495DA37F10>: Failed to establish a new connection: [WinError 10060] A connection attempt failed because the connected party did not properly respond after a period of time, or established connection failed because connected host has failed to respond'))": 1, "can only concatenate str (not \"float\") to str": 12, "HTTPSConnectionPool(host='www.google.com', port=443): Max retries exceeded with url: /finance/quote/ED:NYSE (Caused by NewConnectionError('<urllib3.connection.HTTPSConnection object at 0x000002495DF10A90>: Failed to establish a new connection: [WinError 10060] A connection attempt failed because the connected party did not properly respond after a period of time, or established connection failed because connected host has failed to respond'))": 1, "HTTPSConnectionPool(host='www.google.com', port=443): Max retries exceeded with url: /finance/quote/DB:NYSE (Caused by NewConnectionError('<urllib3.connection.HTTPSConnection object at 0x000002495DEF12E0>: Failed to establish a new connection: [Errno 11001] getaddrinfo failed'))": 1, "HTTPSConnectionPool(host='www.google.com', port=443): Max retries exceeded with url: /finance/quote/6981:TYO (Caused by NewConnectionError('<urllib3.connection.HTTPSConnection object at 0x000002495D3C7910>: Failed to establish a new connection: [WinError 10060] A connection attempt failed because the connected party did not properly respond after a period of time, or established connection failed because connected host has failed to respond'))": 1, "HTTPSConnectionPool(host='www.google.com', port=443): Max retries exceeded with url: /finance/quote/6273:TYO (Caused by SSLError(SSLZeroReturnError(6, 'TLS/SSL connection has been closed (EOF) (_ssl.c:1129)')))": 1, "HTTPSConnectionPool(host='www.google.com', port=443): Max retries exceeded with url: /finance/quote/HLAG:ETR (Caused by NewConnectionError('<urllib3.connection.HTTPSConnection object at 0x000002495D33EEB0>: Failed to establish a new connection: [WinError 10060] A connection attempt failed because the connected party did not properly respond after a period of time, or established connection failed because connected host has failed to respond'))": 1, "HTTPSConnectionPool(host='www.google.com', port=443): Max retries exceeded with url: /finance/quote/DFS:NYSE (Caused by NewConnectionError('<urllib3.connection.HTTPSConnection object at 0x000002495E621430>: Failed to establish a new connection: [WinError 10060] A connection attempt failed because the connected party did not properly respond after a period of time, or established connection failed because connected host has failed to respond'))": 1, "HTTPSConnectionPool(host='www.google.com', port=443): Max retries exceeded with url: /finance/quote/XEL:NASDAQ (Caused by NewConnectionError('<urllib3.connection.HTTPSConnection object at 0x000002495D9A6700>: Failed to establish a new connection: [WinError 10060] A connection attempt failed because the connected party did not properly respond after a period of time, or established connection failed because connected host has failed to respond'))": 1, "HTTPSConnectionPool(host='www.google.com', port=443): Max retries exceeded with url: /finance/quote/600104:SHA (Caused by NewConnectionError('<urllib3.connection.HTTPSConnection object at 0x000002495D9D2940>: Failed to establish a new connection: [WinError 10060] A connection attempt failed because the connected party did not properly respond after a period of time, or established connection failed because connected host has failed to respond'))": 1, "HTTPSConnectionPool(host='www.google.com', port=443): Max retries exceeded with url: /finance/quote/STE:NYSE (Caused by NewConnectionError('<urllib3.connection.HTTPSConnection object at 0x000002495FACE460>: Failed to establish a new connection: [Errno 11001] getaddrinfo failed'))": 1, "HTTPSConnectionPool(host='www.google.com', port=443): Max retries exceeded with url: /finance/quote/LH:NYSE (Caused by NewConnectionError('<urllib3.connection.HTTPSConnection object at 0x000002495FACE460>: Failed to establish a new connection: [Errno 11001] getaddrinfo failed'))": 1, "HTTPSConnectionPool(host='www.google.com', port=443): Max retries exceeded with url: /finance/quote/KB:NYSE (Caused by NewConnectionError('<urllib3.connection.HTTPSConnection object at 0x000002495FACE580>: Failed to establish a new connection: [Errno 11001] getaddrinfo failed'))": 1, "HTTPSConnectionPool(host='www.google.com', port=443): Max retries exceeded with url: /finance/quote/2891:TPE (Caused by NewConnectionError('<urllib3.connection.HTTPSConnection object at 0x000002495FACEA30>: Failed to establish a new connection: [Errno 11001] getaddrinfo failed'))": 1, "HTTPSConnectionPool(host='www.google.com', port=443): Max retries exceeded with url: /finance/quote/TS:NYSE (Caused by NewConnectionError('<urllib3.connection.HTTPSConnection object at 0x000002495FA32370>: Failed to establish a new connection: [Errno 11001] getaddrinfo failed'))": 1, "HTTPSConnectionPool(host='www.google.com', port=443): Max retries exceeded with url: /finance/quote/CINF:NASDAQ (Caused by NewConnectionError('<urllib3.connection.HTTPSConnection object at 0x000002495FA32430>: Failed to establish a new connection: [Errno 11001] getaddrinfo failed'))": 1, "HTTPSConnectionPool(host='www.google.com', port=443): Max retries exceeded with url: /finance/quote/601066:SHA (Caused by NewConnectionError('<urllib3.connection.HTTPSConnection object at 0x000002495E6324C0>: Failed to establish a new connection: [WinError 10060] A connection attempt failed because the connected party did not properly respond after a period of time, or established connection failed because connected host has failed to respond'))": 1, "HTTPSConnectionPool(host='www.google.com', port=443): Max retries exceeded with url: /finance/quote/BAYN:ETR (Caused by NewConnectionError('<urllib3.connection.HTTPSConnection object at 0x000002495FC32FD0>: Failed to establish a new connection: [WinError 10060] A connection attempt failed because the connected party did not properly respond after a period of time, or established connection failed because connected host has failed to respond'))": 1, "HTTPSConnectionPool(host='www.google.com', port=443): Max retries exceeded with url: /finance/quote/DG:NYSE (Caused by SSLError(SSLZeroReturnError(6, 'TLS/SSL connection has been closed (EOF) (_ssl.c:1129)')))": 1, "('Connection aborted.', RemoteDisconnected('Remote end closed connection without response'))": 1, "HTTPSConnectionPool(host='www.google.com', port=443): Max retries exceeded with url: /finance/quote/FM:TSE (Caused by NewConnectionError('<urllib3.connection.HTTPSConnection object at 0x000002495E054E80>: Failed to establish a new connection: [WinError 10060] A connection attempt failed because the connected party did not properly respond after a period of time, or established connection failed because connected host has failed to respond'))": 1, "HTTPSConnectionPool(host='www.google.com', port=443): Max retries exceeded with url: /finance/quote/5401:TYO (Caused by NewConnectionError('<urllib3.connection.HTTPSConnection object at 0x000002495E03E160>: Failed to establish a new connection: [Errno 11001] getaddrinfo failed'))": 1, "HTTPSConnectionPool(host='www.google.com', port=443): Max retries exceeded with url: /finance/quote/AMCR:NYSE (Caused by NewConnectionError('<urllib3.connection.HTTPSConnection object at 0x000002495E03E1F0>: Failed to establish a new connection: [Errno 11001] getaddrinfo failed'))": 1, "HTTPSConnectionPool(host='www.google.com', port=443): Max retries exceeded with url: /finance/quote/VTR:NYSE (Caused by NewConnectionError('<urllib3.connection.HTTPSConnection object at 0x000002495D908580>: Failed to establish a new connection: [WinError 10060] A connection attempt failed because the connected party did not properly respond after a period of time, or established connection failed because connected host has failed to respond'))": 1, "HTTPSConnectionPool(host='www.google.com', port=443): Max retries exceeded with url: /finance/quote/ULTA:NASDAQ (Caused by NewConnectionError('<urllib3.connection.HTTPSConnection object at 0x000002495D8F26D0>: Failed to establish a new connection: [WinError 10060] A connection attempt failed because the connected party did not properly respond after a period of time, or established connection failed because connected host has failed to respond'))": 1}}